Sources/Alloy/Core/Extensions/Metal/MTLDevice+Features.swift

```diff
@@ -15,6 +16,24 @@ public extension MTLDevice {
             #elseif os(macOS)
             return self.supportsFeatureSet(.macOS_GPUFamily1_v3)
             #endif
+            
+        case let .readWriteTexture(pixelFormat):
+            let tierOneSupportedPixelFormats: Set<MTLPixelFormat> = [
+                .r32Float, .r32Uint, .r32Sint
+            ]
+            let tierTwoSupportedPixelFormats: Set<MTLPixelFormat> = tierOneSupportedPixelFormats.union([
+                .rgba32Float, .rgba32Uint, .rgba32Sint, .rgba16Float,
+                .rgba16Uint, .rgba16Sint, .rgba8Unorm, .rgba8Uint,
+                .rgba8Sint, .r16Float, .r16Uint, .r16Sint,
+                .r8Unorm, .r8Uint, .r8Sint
+            ])
+            
+            switch self.readWriteTextureSupport {
+            case .tier1: return tierOneSupportedPixelFormats.contains(pixelFormat)
+            case .tier2: return tierTwoSupportedPixelFormats.contains(pixelFormat)
+            case .tierNone: return false
+            @unknown default: return false
```

Comment:
is it safe to add this? especially with false clause? I think all new cases should correspond to more modern designs

Reply:
im not sure about the desktop GPUs